metaflow/package.py

```diff
@@ -21,14 +21,9 @@ def __init__(self, flow, environment, echo, suffixes=DEFAULT_SUFFIXES_LIST):
         try:
             import metaflow_extensions
         except ImportError:
-            self.metaflow_extensions_root = None
+            self.metaflow_extensions_path = []
         else:
-            self.metaflow_extensions_root = os.path.dirname(
-                metaflow_extensions.__file__
-            )
-            self.metaflow_extensions_addl_suffixes = getattr(
-                metaflow_extensions, "METAFLOW_EXTENSIONS_PACKAGE_SUFFIXES", None
-            )
+            self.metaflow_extensions_path = list(metaflow_extensions.__path__)
```

Comment:
This seems fine but please re-add the PACKAGE_SUFFIXES portion to the else clause.

Reply:
`METAFLOW_EXTENSIONS_PACKAGE_SUFFIXES` seems to be referenced at only this one location in the Metaflow code. Also, Google and GitHub searches for `METAFLOW_EXTENSIONS_PACKAGE_SUFFIXES` only return the removed code.

Is the feature associated with `METAFLOW_EXTENSIONS_PACKAGE_SUFFIXES` documented anywhere? Is this used / needed by anything? If yes, I could add a corresponding comment.

Reply:
Good point but yes, it is used in the extensions mechanism. In our case, we actually package a `.so` with metaflow (for a not-as-yet released feature of Metaflow) and so we set this variable in our extension package.

Reply:
Ok, thanks for explaining this point. I will add re-introduce the corresponding lines of code.

Reply:
Done in add3c40